.github/workflows/build.yml

```diff
@@ -1,13 +1,13 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# This workflow will do a clean installation of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
 name: build
 
 on:
   push:
-    branches: [main ]
+    branches: [ main, dev, certification ]
```

Comment:
This is redundant. There is no dev and certification branches in utils

Reply:
fixed